File: MyStack/MyStack/EntryPoint.cs
using System;

namespace MyStack
{
  /// <summary>
  /// The entry point of the programm
  /// </summary>
  class EntryPoint
  {
    /// <summary>
    /// Calls all methods
    /// </summary>
    /// <param name="args">arguments from console</param>
    static void Main(string[] args)
    {
      try
      {
        StackOnList<int> myStack = new StackOnList<int>();
        myStack.Push(1);
        myStack.Push(2);
        myStack.Push(3);
        myStack.Push(4);

        myStack.Pop();
        myStack.Peek();
      }
      catch(Exception ex)
      {
        Console.Write(ex.Message);
      }
    }
  }
}

File: test_06/src/main/java/EntryPoint.java
/**
 * This class is entry point.
 */
public class EntryPoint {
  public static void main(String[] args) {
    MyStack stack = new MyStack();
    stack.push("First" );
    stack.peek();
    System.out.println("Empty status = " + stack.isEmpty());
    System.out.println("Stack size : " + stack.getSize());
  }
}

